bot/exts/evergreen/snakes/_utils.py

```diff
@@ -55,7 +55,8 @@
     "Baby Rattle Snake": "https://i.imgur.com/i5jYA8f.png",
     "Baby Dragon Snake": "https://i.imgur.com/SuMKM4m.png",
     "Baby Garden Snake": "https://i.imgur.com/5vYx3ah.png",
-    "Baby Cobra": "https://i.imgur.com/jk14ryt.png"
+    "Baby Cobra": "https://i.imgur.com/jk14ryt.png",
+    "Baby Anaconda": "https://i.imgur.com/4zT4aij.png",
```

Comment:
The Baby Anaconda is very different from the other icons here, we should probably look into where the other pictures were found so that we get a uniform front.

Reply:
Agreed, it looks a bit out of place. I've had a look to try and find the origin of the others but can't find anything. One alternative would be to continue the joke with the baby python option showing the python logo, and show the [anaconda logo](https://upload.wikimedia.org/wikipedia/en/c/cd/Anaconda_Logo.png) instead. Thoughts?

Reply:
Thats a really great idea, showing the anaconda logo however the python logo kinda looks like a snake but the anaconda logo is like a circle. Photoshop the 2 maybe ?? or the anaconda logo works as well!, let me know after which i can commit the changes